packages/data/addon/adapters/facts/interface.ts

```diff
@@ -116,17 +116,46 @@ export type AsyncQueryResponse = {
   };
 };
 
+export interface TableExport {
```

Comment:
Is this type being used?

Reply:
Will remove AsyncQuery from here as well